plugin/src/main/java/org/opensearch/sql/plugin/transport/TransportSQLQueryAction.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.plugin.transport;
+
+import org.opensearch.action.ActionListener;
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.support.ActionFilters;
+import org.opensearch.action.support.HandledTransportAction;
+import org.opensearch.client.Client;
+import org.opensearch.common.inject.Inject;
+import org.opensearch.commons.sql.action.SQLActions;
+import org.opensearch.commons.sql.action.TransportSQLQueryRequest;
+import org.opensearch.commons.sql.action.TransportSQLQueryResponse;
+import org.opensearch.tasks.Task;
+import org.opensearch.transport.TransportService;
+import org.opensearch.commons.utils.TransportHelpersKt;
+
+import java.io.IOException;
+
+/**
+ * Send SQL query transport action
+ */
+public class TransportSQLQueryAction extends HandledTransportAction<ActionRequest, TransportSQLQueryResponse> {
```

Comment:
In PR, https://github.com/opensearch-project/common-utils/pull/155/,  TransportQueryRequest is defined, instead of TransportSQLQueryAction

Reply:
@penghuo Hi Peng, sorry I forget to submitted the commit to common-utils after the change, could you take a look again? I separated `TransportQueryRequest` into `TrasnportSQLQueryRequest` and `TransportPPLQueryRequest`  https://github.com/opensearch-project/common-utils/pull/155